sockeye/encoder.py

```diff
@@ -93,10 +93,16 @@ def get_max_seq_len(self) -> Optional[int]:
 
 class FactorConfig(config.Config):
 
-    def __init__(self, vocab_size: int, num_embed: int) -> None:
+    def __init__(self,
+                 vocab_size: int,
+                 num_embed: int,
+                 combine: str, # From C.SOURCE_FACTORS_COMBINE_CHOICES
```

Comment:
any value in making this an Enum?

Reply:
It's a bit cleaner, although the whole parameter validation is already performed by argparse.